addons/purchase_requisition/models/purchase.py

```diff
@@ -286,6 +286,10 @@ def _compute_price_unit_and_date_planned_and_name(self):
                         date=pol.order_id.date_order and pol.order_id.date_order.date(),
                         uom_id=line.product_uom_id,
                         params=params)
+
+                    if pol or not pol.date_planned:
```

Comment:
Not sure why it would be `pol`, did you mean `seller` so that it copies the pre-v16 behavior?
```suggestion
                    if seller or not pol.date_planned:
```

Reply:
Indeed, my bad

Reply:
On second though, maybe only `if not pol.date_planned` makes more sense since when the PO is first created, the date_planned is passed during the POL creation and we don't want to overwrite it by accident